pkg/local_object_storage/metabase/ec.go

```diff
@@ -82,7 +82,7 @@ func (db *DB) resolveECPartWithPayloadLenTx(tx *bbolt.Tx, cnr cid.ID, parent oid
 
 	metaBktCrs := metaBkt.Cursor()
 
-	id, err := db.resolveECPartInMetaBucket(metaBktCrs, cnr, parent, pi)
+	id, err := db.resolveECPartInMetaBucket(metaBktCrs, cnr, parent, pi, false)
```

Comment:
This different in behavior is not something I'd expect between (,WithPayloadLen) external functions. And it should be documented then.

Reply:
i only touched on the code involved in the changeable behavior. Agree about docs, will upd them